src/System.Runtime.Loader/tests/DefaultContext/DefaultLoadContextTest.cs

```diff
@@ -183,7 +183,7 @@ public static void LoadNonExistentInDefaultContext()
                 () => AssemblyLoadContext.Default.LoadFromAssemblyName(new AssemblyName("System.Runtime.Loader.NonExistent.Assembly")));
         }
 
-        public void DefaultContextFallback()
+        private void DefaultContextFallback()
```

Comment:
Why are these being marked private instead of `[Fact]`?

Reply:
Because these are helper methods which are invoked in the test method above.